src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -1415,7 +1415,15 @@ function setupFsCopyModal() {
                 for (var i = 0 ; i < coursesTable.length; i++) {
                     htmlToAppend += "<div class=\"checkbox\">";
                     htmlToAppend += "<label><input type=\"checkbox\" name=\"copiedcoursesid\"";
-                    htmlToAppend += "value=\"" + coursesTable[i].id + "\"> [" + coursesTable[i].id + "] : " + coursesTable[i].name;
+                    if (String(coursesTable[i].id) == courseid){
+                    	htmlToAppend += "value=\"" + coursesTable[i].id + "\"> [" + "<font color=\"red\">" + coursesTable[i].id + "</font>" + "] : " + coursesTable[i].name;
```

Comment:
Instead of using the "font" html element, which is obsolete (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/font), it might be better to just use an existing css class. We have a css class "text-color-red" that you can use here.